homework/input-output/src/main/java/com/cbfacademy/App.java

```diff
@@ -1,7 +1,25 @@
 package com.cbfacademy;
 
+import java.util.HashMap;
+import java.util.Properties;
+
 public class App {
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        String existingFilePath = "homework/input-output/src/test/resources";
+
+        HashMap<String, String> propMap = new HashMap<String, String>() {
+            {
+                put("db.url", "localhost");
+                put("db.port", "5353");
+                put("db.user", "cbfacademy");
+                put("db.password", "password");
```

Comment:
Should the keys and values match:

```
datasource.url=jdbc:postgresql://localhost:port_no/db_name
datasource.username=postgres
datasource.password=password
```

Reply:
Heya, this is separate exercise. There's one step where they need to read from an existing file (`example.properties`) and another step where they need to create a new file with given values (`configuration.properties`)